Design.md

```diff
@@ -174,6 +174,13 @@ Originally discussed in [#12](https://github.com/coreos/fedora-coreos-tracker/is
  - For the short term, if we need to include an agent we will bake it into the image. We will not have any specific
    mechanism for including agents.
 
+### AWS:
+
+Originally discussed in [#66](https://github.com/coreos/fedora-coreos-tracker/issues/66).
+
+- AWS does not require a cloud agent but does require NVME EBS udev rules and a script located at [coreos/init#268](https://github.com/coreos/init/pull/268)
```

Comment:
I'd say let's leave out the reference to `https://github.com/coreos/init/pull/268`. so just end this `EBS udev rules and a script`

the reference to #104 below should be enough

Reply:
My editor automatically removed the whitespace so I manually added them back in with another editor. I also addressed the comment :arrow_up: .  Something is up with my branch but I don't think it will affect the PR or the git history.